cluster/gce/util.sh

```diff
@@ -646,6 +577,7 @@ function kube-up {
   for (( i=0; i<${#MINION_NAMES[@]}; i++)); do
     create-route "${MINION_NAMES[$i]}" "${MINION_IP_RANGES[$i]}" &
     add-instance-metadata "${MINION_NAMES[$i]}" "node-ip-range=${MINION_IP_RANGES[$i]}" &
+    add-instance-metadata "${MINION_NAMES[$i]}" "node-name=${MINION_NAMES[$i]}" &
```

Comment:
This won't work, is racy if you have a lot of nodes. You need to use the multi-KV form of `add-instance-metadata` that does it in one GCE command, otherwise the command may fail.